latex/Research_Strategy.tex

```diff
@@ -138,27 +138,29 @@
+Despite this progress, these methods currently have severe limitations, and a great deal of science and engineering is still needed before these techniques can achieve their full potential in transforming molecular design.
+For example, even ``small'' protein conformational conformational changes not gracefully handled by current methodologies can yield errors up to 5 kcal/mol in calculated binding free energies~\cite{lim_sensitivity_2016}, force field limitations still pose major challenges~\cite{rocklin_blind_2013}, and the inability to treat important chemical effects like protonation and tautomer equilibria drastically limits the domain of applicability.
+For many pharmaceutically relevant systems, the most important sources of error are not yet clear.
+
+Progress on addressing these challenges has been frustratingly slow, hindered by a lack of high-quality data and community focus.
+{\bf Neither retrospective tests nor propsective application in active drug discovery projects provides the necessary impetus and data to rapidly advance physical modeling techniques to overcome existing major barriers to widespread utility.}
+While large-scale retrospective tests can be valuable for assessing how well we can currently do \emph{retrospectively}, they are of limited value for paving the way forward or for assessing actual utility for prospective molecular design.
+Partly, retrospective tests allow for unintentional statistical transgressions, where a well-meaning researcher might apply a protocol known to work in a particular situation to achieve good statistical performance, but this reported retrospective performance is unpredictive of general prospective performance.
+For example, if the binding mode of a ligand is already known crystallographically, a researcher may use that binding mode in retrospective tests, whereas prospective or design work would require first selecting among candidate binding modes, introducing substantial uncertainty unaccounted for in the retrospective statistics~\cite{mobley_predicting_2007, boyce_predicting_2009, mobley_perspective_2012}.
 This also means that in retrospective tests, researchers almost invariably try far fewer methods than in prospective tests, resulting in much less new insight.
-Prospective application, while important, also does not provide the necessary impetus, partly because often, the predicted compounds are in fact never tested~\cite{christ_binding_2016} or the experimental data necessary to assess the quality of the predictions is absent -- for example, because binding affinities are not measured or no crystallography is available. 
+{\color{red}[JDC: Not sure I buy that---you can easily overfit in retrospective tests by trying many approaches to improve your score.]}
```

Comment:
Totally agree. I was making a different point, and probably there is a better way to say it, so I could use your help. The point I was making is this -- if you already know that a particular system is "well behaved", you know the relevant binding modes, etc., it's easy to just say, "oh, well, we don't need to worry about THOSE issues here" and so people often apply rather simple protocols in retrospective tests. Sure, they might apply a variety of simple variations of those protocols until they get the right answer, thereby overfitting, as you're pointing out. But, they tend to take a simple, straightforward approach.

What we tend to see in SAMPL, instead, is that people try a whole bunch of more complicated protocols because they are so uncertain about what the issues/challenges might even be. (People may try different binding modes of a ligand, or different protonation states, or different charge sets, etc.)

Anyway, that's more detail than we need to get into here. Would love thoughts on the best way to sell why prospective is so much better than retrospective, clearly. (Even Robert Abel is not really convinced of this.)